README.md

```diff
@@ -20,8 +20,48 @@ which stores the key for future access. The provided storage strategy
 implementation encrypts the symmetric key using a public/private key pair and
 then stores the encrypted key on the `FileSystem` with the encrypted file.
 
-Example
--------
+Standalone Example
+------------------
+
+The hadoop-crypto-all.jar can be added to the classpath of any client and used
+to wrap any concrete backing FileSystem. The scheme of the EncryptedFileSystem
+is `e[FS-scheme]` where `[FS-scheme]` is any FileSystem that can be instantiated
+statically using `FileSystem#get` (eg: efile). The FileSystem implementation,
+public key, and private key must be configured in the core-site.xml as well.
+
+### Hadoop Cli
+
+Add hadoop-crypto-all.jar to the classpath of the cli (ex: share/hadoop/common).
+
+##### core-site.xml
+```xml
+<configuration>
+    <property>
+        <name>fs.efile.impl</name> <!-- others: fs.es3a.impl or fs.ehdfs.impl -->
+        <value>com.palantir.hadoop.ConfigurableEncryptedFileSystem</value>
+    </property>
+
+    <property>
+        <name>fs.efs.key.public</name>
+        <value>REDACTED</value>
```

Comment:
I see you mention the format below. Would still be good to give an example and show how to generate then in linux